CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Add missing OrderEvents during checkout flow - #5684 by @koradon
 - Update google merchant to get tax rate based by plugin manager - #5823 by @gabmartinez
 - Allow unicode in slug fields - #5877 by @IKarbowiak
+- Anonymize and update order and payment fields - #5926 by @IKarbowiak
```

Comment:
I suggest moving this to "Breaking changes", since this PR removes a mutation and some fields. We could actually mention which mutation and fields were removed.